packages/create-block-tutorial-template/plugin-templates/$slug.php.mustache

```diff
@@ -30,18 +30,14 @@
  * @package           {{namespace}}
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
```

Comment:
This is the most recent addition to all entry PHP files. See the full explanation in https://github.com/WordPress/gutenberg/pull/55533.

Reply:
Good catch. Fixed!